<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetImporter</class>
 <widget class="QDialog" name="DatasetImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Dataset</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCommandLinkButton" name="loadFolder">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Load dataset from a folder</string>
       </property>
       <property name="statusTip">
        <string>Load dataset from a folder</string>
       </property>
       <property name="text">
        <string>Load From Folder</string>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dataset Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="datasetName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Species (Optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="species"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Tissue (Optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tissue"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Comments (Optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="comments"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Chip size</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="chip_x">
         <property name="minimum">
          <number>33</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="chip_y">
         <property name="minimum">
          <number>35</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ST Data file</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="stDataFile"/>
       </item>
       <item>
        <widget class="QToolButton" name="loadSTDataFile">
         <property name="cursor">
          <cursorShape>SplitHCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Choose file</string>
         </property>
         <property name="statusTip">
          <string>Choose file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Alignment file (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="imageAlignmentFile"/>
       </item>
       <item>
        <widget class="QToolButton" name="loadImageAlignmentFile">
         <property name="cursor">
          <cursorShape>SplitVCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Choose file</string>
         </property>
         <property name="statusTip">
          <string>Choose file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Spot mapping file (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="spotMapFile"/>
       </item>
       <item>
        <widget class="QToolButton" name="loadSpotMapFile">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Choose file</string>
         </property>
         <property name="statusTip">
          <string>Choose file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Main image</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="mainImageFile"/>
       </item>
       <item>
        <widget class="QToolButton" name="loadMainImageFile">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Choose file</string>
         </property>
         <property name="statusTip">
          <string>Choose file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>datasetName</zorder>
  <zorder>label_7</zorder>
  <zorder>species</zorder>
  <zorder>label_9</zorder>
  <zorder>tissue</zorder>
  <zorder>label_8</zorder>
  <zorder>comments</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>buttonBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_12</zorder>
  <zorder>label_10</zorder>
 </widget>
 <tabstops>
  <tabstop>datasetName</tabstop>
  <tabstop>species</tabstop>
  <tabstop>tissue</tabstop>
  <tabstop>comments</tabstop>
  <tabstop>loadSTDataFile</tabstop>
  <tabstop>mainImageFile</tabstop>
  <tabstop>loadMainImageFile</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatasetImporter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatasetImporter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
